Directory.Build.props

```diff
@@ -29,4 +30,14 @@
     <SdkLayoutDirectory>$(ArtifactsBinDir)$(Configuration)\dotnetLayout</SdkLayoutDirectory>
   </PropertyGroup>
 
+  <Target Name="OverrideSymbolsOutput" BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
+          Condition="'$(DebugType)' != 'embedded'">
+
+    <ItemGroup>
+      <_DebugSymbolsOutputPath Remove="@(_DebugSymbolsOutputPath)" />
+      <_DebugSymbolsOutputPath Include="@(_DebugSymbolsIntermediatePath->'$(ArtifactsSymStoreDirectory)$(MSBuildProjectName)\$(TargetFramework.ToLowerInvariant())\%(Filename)%(Extension)')" />
+    </ItemGroup>
```

Comment:
Why do we need to do anything special. Doesn't arcade handle this automatically?